images/ceph/Makefile

```
@@ -104,6 +110,11 @@ generate-csv-templates: $(OPERATOR_SDK) $(YQ) ## Generate CSV templates for OLM
 	@OLM_CATALOG_DIR=$(CSV_TEMPLATE_DIR)/cluster/olm/ceph ../../cluster/olm/ceph/generate-rook-csv-templates.sh
 	@echo " === Generated CSV templates can be found at $(CSV_TEMPLATE_DIR)/cluster/olm/ceph/templates"
 
+get-volume-replication-crds:
+	@if [[ -z "$(CURL_TEMPLATE_DIR)" ]]; then echo "CURL_TEMPLATE_DIR is not set"; exit 1; fi
+	curl $(VOL_REPL_URL)/$(VOLUME_REPLICATION_CRD) -o $(CURL_TEMPLATE_DIR)/$(VOLUME_REPLICATION_CRD)
```

```suggestion
	curl -L $(VOL_REPL_URL)/$(VOLUME_REPLICATION_CRD) -o $(CURL_TEMPLATE_DIR)/$(VOLUME_REPLICATION_CRD)
```

Typically safer if we get a redirect.

---

Where is. `CURL_TEMPLATE_DIR` coming from?